### PR TITLE
Added user location marker

### PR DESCRIPTION
**Pull request by Albermonte**
The marker only appears if the user clicks on the location button and the accuracy is lower than 1000m

The marker radius depends also in the accuracy (between 64px and 24px)

This is how it looks
image

### CODE REVIEW

**Comment by onmax** [comment]
Noice, before merging I would like to make the radius dynamic to zoom and small fixes

**Comment by onmax** [comment]
Todo add localize myself in compact view

**Comment by onmax** [comment]
@Albermonte ready to merge, what do you think?

File: apps/web/src/components/UserLocationMarker.vue [hunk 1]
```diff
@@ -1,12 +1,23 @@
 <script lang="ts" setup>
-const props = defineProps<{ accuracy: number, zoom: number }>()
+import { EstimatedMapPosition } from "types";
+import { metersToPx } from "../../../../packages/geo/src/geo-utils.js";
```
Comment (Albermonte) [inline apps/web/src/components/UserLocationMarker.vue]: This looks ugly, I think you can use `@packages/geo...` or similar

File: apps/web/src/components/MapMarkers.vue [hunk 2]
```diff
@@ -3,14 +3,14 @@ const { singles } = storeToRefs(useMarkers())
...
-const { browserPositionIsSupported, browserPosition } = useGeoIp()
+const { browserPositionIsSupported, browserPosition } = storeToRefs(useGeoIp())
 </script>
 
 <template>
    <!-- User Position Marker -->
   <CustomMarker
-    v-if="browserPositionIsSupported && browserPosition && browserPosition.accuracy < 1000 && browserPosition.accuracy > 0"
-    :options="{ position: { lng: browserPosition.center.lng, lat: browserPosition.center.lat } }" data-custom-marker
+    v-if="browserPositionIsSupported && browserPosition.accuracy > 0 && browserPosition?.accuracy < 1000"
```
Comment (Albermonte) [inline apps/web/src/components/MapMarkers.vue]:
```suggestion
    v-if="browserPositionIsSupported && browserPosition?.accuracy > 0 && browserPosition?.accuracy < 1000"
```

**Review by Albermonte — COMMENTED** [review]
Cannot test the changes because it's giving me this error

image

**Comment by onmax** [comment]
i guess in summary: skill issues from you

File: apps/web/src/components/UserLocationMarker.vue (hunk 1 above)
Comment (onmax) [reply apps/web/src/components/UserLocationMarker.vue]: yep

**Comment by onmax** [comment]
>Cannot test the changes because it's giving me this error

now it builds again

**Pull request merged**